webplugin/css/app/mck-sidebox-1.0.css

```diff
@@ -6231,4 +6231,79 @@ body:not(.accesibility) :focus {
 }
 body.accesibility :focus {
 	outline: 5px auto #3b99fc;
-}
\ No newline at end of file
+}
+
+/* Chat Popup Template Code Starts */
+
+.chat-popup-widget-container {
+    box-shadow: 0 2px 9px 2px rgba(0, 0, 0, 0.17);
+    background-color: #ffffff;
+    position: absolute;
+	min-height: 70px;
+	font-family: 'Roboto', sans-serif;
+	width: 100%;
+	max-width: 380px;
+	cursor: pointer;
+}
+
+.chat-popup-widget-container--horizontal {
+	border-radius: 40px;
+	bottom: 5px;
+	right: 5px;
+}
+
+.chat-popup-widget-container--vertical {
+	border-radius: 10px;
+    bottom: 7px;
+	right: 85px;
+	min-height: 75px;
+	max-width: 300px;
+}
+
+.chat-popup-widget-text-wrapper {
+
```

Comment:
any specific reason for adding an empty class?

Reply:
@akshatdhabi Yes.

Reply:
@CaptainT33mo  please elaborate.